_maps/RandomRuins/SpaceRuins/deepstorage.dmm

```diff
@@ -1,421 +1,6873 @@
-"eF" = (/obj/machinery/atmospherics/pipe/simple/supplymain/hidden,/obj/machinery/light/small{dir = 8},/turf/open/floor/plasteel/floorgrime{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage)
-"eG" = (/obj/machinery/atmospherics/pipe/simple/orange/hidden{tag = "icon-intact (SOUTHEAST)"; icon_state = "intact"; dir = 6},/turf/open/floor/plasteel/floorgrime{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage)
-"eH" = (/obj/machinery/atmospherics/pipe/simple/orange/hidden{tag = "icon-intact (EAST)"; icon_state = "intact"; dir = 4},/turf/closed/wall/mineral/iron{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage/airlock)
-"eI" = (/obj/structure/table,/obj/machinery/button/door{id = "bunkerexterior"; name = "exterior blastdoor access"; pixel_x = -24; pixel_y = -8; req_access_txt = "200"},/obj/machinery/button/door{id = "bunkerinterior"; name = "interior blastdoor access"; pixel_x = -24; pixel_y = 0; req_access_txt = "200"},/obj/machinery/button/door{id = "bunkershutter"; name = "hallway shutter toggle"; pixel_x = -24; pixel_y = 8; req_access_txt = "200"},/obj/machinery/atmospherics/pipe/simple/orange/hidden{tag = "icon-intact (EAST)"; icon_state = "intact"; dir = 4},/obj/machinery/computer/security/telescreen{dir = 1; name = "Bunker Entrance"; network = list("Bunker1"); pixel_x = 0; pixel_y = 2},/turf/open/floor/plasteel/floorgrime{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage/airlock)
-"eJ" = (/obj/machinery/atmospherics/pipe/simple/orange/hidden{tag = "icon-intact (EAST)"; icon_state = "intact"; dir = 4},/obj/machinery/atmospherics/pipe/simple/supplymain/hidden,/obj/structure/cable/yellow{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/structure/cable/yellow{icon_state = "1-4"; d1 = 1; d2 = 4},/turf/open/floor/plasteel/floorgrime{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage/airlock)
-"eK" = (/obj/machinery/atmospherics/pipe/simple/orange/hidden{tag = "icon-intact (EAST)"; icon_state = "intact"; dir = 4},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/obj/structure/cable/yellow{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_y = 0},/obj/machinery/firealarm{pixel_y = 24},/turf/open/floor/plasteel/floorgrime{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage/airlock)
-"eL" = (/obj/machinery/atmospherics/pipe/simple/orange/hidden{tag = "icon-intact (SOUTHWEST)"; icon_state = "intact"; dir = 10},/obj/structure/cable/yellow{d2 = 8; icon_state = "0-8"},/obj/machinery/power/apc{dir = 1; name = "Airlock Control APC"; pixel_x = 0; pixel_y = 24},/turf/open/floor/plasteel/floorgrime{baseturf = /turf/open/floor/plating/asteroid/airless},/area/ruin/deepstorage/airlock)
-"eM" = (/obj/structure/grille,/obj/structure/window/reinforced/highpressure{tag = "icon-rwindow (WEST)"; icon_state = "rwindow"; dir = 8},/obj/structure/window/reinforced/highpressure{tag = "icon-rwindow (EAST)"; ico
```

Comment:
Uh, these aren't actually connected. Need to change d1 = 4; to d1 =0; (or remove it since it defaults to 0) on both of them, otherwise they're considered angled wires.